frontend/src/scenes/insights/InsightTabs/TrendTab/TrendTab.tsx

```diff
@@ -31,10 +31,10 @@ export function TrendTab({ view }: TrendTabProps): JSX.Element {
     const { groupsTaxonomicTypes } = useValues(groupsModel)
     const [isUsingFormulas, setIsUsingFormulas] = useState(filters.formula ? true : false)
     const lifecycles = [
-        { name: 'new', tooltip: 'Users that are new.' },
-        { name: 'resurrecting', tooltip: 'Users who were once active but became dormant, and are now active again.' },
-        { name: 'returning', tooltip: 'Users who consistently use the product.' },
-        { name: 'dormant', tooltip: 'Users who are inactive.' },
+        { name: 'new', tooltip: 'Users who signed up and did activity during period.' },
+        { name: 'returning', tooltip: 'Users who did activity both this and previous period.' },
```

Comment:
```suggestion
        { name: 'returning', tooltip: 'Users who did the activity in both this and the previous period. Users signed up before this period.' },
```

Reply:
I think referring to user sign ups outside new just makes everything confusing to follow - we'd need to mention it in resurrecting as well then.

Reply:
With fresh eyes I also see another problem with mentioning sign ups. It's not exactly accurate. We take the first event for the user, which might not necessarily be a sign up (depending on how our users determine this). Amending my proposal.

```suggestion
        { name: 'returning', tooltip: 'Users who did the activity in both this and the previous period.' },
```